src/js/components/Anchor/__tests__/__snapshots__/Anchor-test.tsx.snap

```diff
@@ -2048,7 +2048,7 @@ exports[`Anchor warns about invalid icon render 1`] = `
   font-weight: 600;
   text-decoration: none;
   cursor: pointer;
-  padding: 12px;
+  padding: small;
```

Comment:
does this look correct @taysea

Reply:
I just looked through our other snapshots and I couldn't find any other places where this happens. It seems like in the snapshots things are always translated into px values, so it seems like something is off here

Reply:
ah okay so yeah we might have to convert it using edge size or something